Makefile

```diff
@@ -69,4 +69,9 @@ pb:
 		.;
 	cp -a output/bess_pb ${BESS_PB_DIR}
 
-.PHONY: docker-build docker-push output pb
+test:
+	DOCKER_BUILDKIT=$(DOCKER_BUILDKIT) docker build $(DOCKER_PULL) $(DOCKER_BUILD_ARGS) \
```

Comment:
I think it's counterintuitive to use Docker build for running tests. + we produce an unnecessary Docker image. I propose to use `golang:1.13` image and use `docker run`. The [Antrea](https://github.com/antrea-io/antrea/blob/main/Makefile#L146) project is a good example.

Reply:
makes sense